### ISSUE TITLE
Bug: studio framerate/performance drop when selecting any character

### ISSUE THREAD

**Issue by johnrose433** [Eusth/PlayClubVR#22]
Excellent mod, as always. Just logging a bug in studio:

1) launch any scene in PlayClubStudioVR.exe and highlight any character in sthe Scene-->Character -->Current Character menu

2) While the character name is highlighted, framerate/performance drops dramatically

3) Deselect the character by clicking on its name again in the same menu (will no longer be highlighted)

4) framerate/performance will go back to normal

**Comment by Eusth** [comment]
Yup, I already noticed this behavior and took a brief look at it, but couldn't find any clues.
For the time being, my assumption is that Illusion's code doesn't perform properly.
